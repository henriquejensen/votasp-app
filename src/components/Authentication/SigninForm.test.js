import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import SigninForm from './SigninForm';

describe('SigninForm', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(( <SigninForm showLoginPage={jest.fn()} /> )))).toMatchSnapshot();
  })
})
