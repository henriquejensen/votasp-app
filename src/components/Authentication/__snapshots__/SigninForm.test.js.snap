// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninForm should match a snapshot 1`] = `
<FormLayout
  activeTab="signin"
  errorMessage={null}
  showLoginPage={[MockFunction]}
>
  <form
    onSubmit={[Function]}
  >
    <div
      className="authentication__form-content"
    >
      <SocialLogin
        updateErrorMessage={[Function]}
      />
      <p
        className="authentication__separator"
      >
        ou
      </p>
      <div
        className="field-wrapper"
      >
        <label
          htmlFor="email"
        >
          E-mail
        </label>
        <input
          className="input"
          id="email"
          name="email"
          required={true}
          type="text"
        />
      </div>
      <div
        className="field-wrapper"
      >
        <label
          htmlFor="password"
        >
          Senha
        </label>
        <input
          className="input"
          id="password"
          name="password"
          required={true}
          type="password"
        />
      </div>
      <Link
        replace={false}
        to="/app/redefinepassword"
      >
        Esqueci minha senha
      </Link>
    </div>
    <button
      className="authentication__submit-button"
    >
      Entrar
    </button>
  </form>
</FormLayout>
`;
